static/css/devhub/new-landing/common.less

```diff
@@ -152,9 +196,17 @@
   background-repeat: no-repeat;
   background-position: 50% 10px;
 
+  html[dir=rtl] & {
+    background-position: 50% 10px;
```

Comment:
isn't that identical to the default, non rtl version? Shouldn't that be reversed?